dateparser_cli/utils.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import os
+from pathlib import Path
+import logging
+
+DEFAULT_UNIX_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", '~/.cache')
+DEFAULT_WIXDOWS_CACHE_DIR = os.environ.get("DEFAULT_CACHE_DIR", os.path.join(str(Path.home()), "AppData", "Roaming"))
+DEFAULT_DIR_NAME = os.environ.get("DEFAULT_DIR_NAME", 'date_parser_models')
+
+date_parser_model_home = None
+
+if sys.platform.startswith('win'):
+    # For Windows :
+    date_parser_model_home = os.path.expanduser(
+        os.path.join(
+            DEFAULT_WIXDOWS_CACHE_DIR, DEFAULT_DIR_NAME
+        )
+    )
+else:
+    # UNIX & OS X :
+    date_parser_model_home = os.path.expanduser(
+        os.path.join(
+            DEFAULT_UNIX_CACHE_DIR, DEFAULT_DIR_NAME
+        )
+    )
```

Comment:
This can be refactored in this way:


```suggestion
if sys.platform.startswith('win'):
    # For Windows
    cache_dir = DEFAULT_WIXDOWS_CACHE_DIR
else:
    # UNIX & OS X
    cache_dir = DEFAULT_UNIX_CACHE_DIR

date_parser_model_home = os.path.expanduser(
    os.path.join(cache_dir, DEFAULT_DIR_NAME)
)
```

Reply:
Thanks, I have made the changes 👍🏻